### PR TITLE
Properly start aw-server before running integration tests

### PR DESCRIPTION
**Pull request by johan-bjareholt**

### CODE REVIEW

**Comment by johan-bjareholt** [comment]
Works fine locally now, waiting for travis

File: aw-client-rust/tests/test.rs [hunk 1]
```diff
@@ -1,13 +1,56 @@
...
+        // TODO: Properly shutdown
+        use aw_server::endpoints::ServerState;
+        let state = ServerState {
+            // TODO: This currently has legacy_import enabled so it can become very slow on
+            // machines with a legacy database on them if it's large
+            datastore: Mutex::new(aw_datastore::Datastore::new_in_memory(false)),
+            asset_path: PathBuf::from("."), // webui won't be used, so it's invalidly set
+        };
+        let aw_config = aw_server::config::AWConfig::default();
+        let server = aw_server::endpoints::build_rocket(state, &aw_config);
```
Comment (ErikBjare) [inline aw-client-rust/tests/test.rs]: Could you use a non-default port?

Comment (johan-bjareholt) [reply aw-client-rust/tests/test.rs]: Done

**Comment by ErikBjare** [comment]
Looks like the build stalls on macOS while packing the cache? I've restarted them for good measure.

On Android it seems to fail due to a change in the function signature of `Database::new`, should be an easy fix.

**Comment by johan-bjareholt** [comment]
Fixed the android issue but the macOS builds still get stuck, merging anyway

**Pull request merged**